src/main/java/bio/terra/service/auth/iam/IamProviderInterface.java

```diff
@@ -260,6 +260,46 @@ String getPetToken(AuthenticatedUserRequest userReq, List<String> scopes)
    */
   String getGroup(String accessToken, String groupName) throws InterruptedException;
 
+  /**
+   * List the members of a DAC group on a SnapshotAccessRequest
```

Comment:
```suggestion
   * List the members of a Sam group
```
These calls are generic to Sam, so they could be used to list any Sam group, not just DAC groups for SnapshotAccessRequests